backend/django_core/static_src/components/page/explore/Index.svelte

```diff
@@ -28,84 +28,64 @@
 
     let search_promise: Promise<Anime[]> | null = null;
 
-
-    onMount(async () => (search_promise = get_anime_with_serach_parameters()));
+    onMount(async () => {
+        search_promise = get_anime_with_serach_parameters();
+    });
 
     // Mapping
     let filter_options_mapping: {
         [key: string]: {
             title: string;
             class: string;
             value: string;
-            items: Record<string, string> | null;
+            items?: Array<string | number>;
             selected_items: string[] | null;
         };
     } = {
         time_range: {
             title: "Time Range",
             class: "hidden flex-col md:gap-[0.35vw]",
             value: "",
-            items: {},
             selected_items: []
         },
         genres: {
             title: "Genres",
             class: "md:flex flex-col md:gap-[0.35vw]",
             value: "",
-            items: {
-                action: "Action",
-                adventure: "Adventure",
-                hentai: "Hentai",
-                romance: "Romance"
-            },
+            items: ["Action", "Adventure", "Hentai", "Romance"],
             selected_items: []
         },
         year: {
             title: "Year",
             class: "md:flex flex-col md:gap-[0.35vw]",
             value: "",
-            items: {
-                2023: "2023",
-                2022: "2022",
-                2021: "2021",
-                2020: "2020"
-            },
+            items: [2020, 2021, 2022, 2023],
             selected_items: []
         },
         season: {
             title: "Season",
             class: "md:flex flex-col md:gap-[0.35vw]",
             value: "",
-            items: {
-                winter: "Winter",
-                spring: "Spring",
-                summer: "Summer",
-                fall: "Fall"
-            },
+            items: ["winter", "spring", "summer", "winter"],
```

Comment:
These duplicate might cause issues while filtering.
Could you please give atleast 1% of FISH focus? :\

Reply:
> These duplicate might cause issues while filtering.
Could you please give atleast 1% of FISH focus? :\

OH, that was a challenge for your eyes. Good thing you caught it